src/components/Reports/HealthIndicatorReport/HealthIndicatorWidget/HealthIndicatorWidget.tsx

```diff
@@ -0,0 +1,157 @@
+import NextLink from 'next/link';
+import React, { Dispatch, SetStateAction, useEffect } from 'react';
+import {
+  Box,
+  Button,
+  CardActions,
+  CardContent,
+  CardHeader,
+  Grid,
+  Skeleton,
+  Typography,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+import AnimatedCard from 'src/components/AnimatedCard';
+import StyledProgress from 'src/components/StyledProgress';
+import { Maybe } from 'src/graphql/types.generated';
+import { useHealthIndicatorWidgetQuery } from './HealthIndicatorWidget.generated';
+
+const StyledCardContent = styled(CardContent)(({ theme }) => ({
+  padding: theme.spacing(2),
+  paddingBottom: 0,
+  height: 'calc(100% - 103px)',
+}));
+
+const StyledBox = styled(Box)(() => ({
+  display: 'flex',
+  gap: 2,
+  justifyContent: 'space-between',
+  alignItems: 'center',
+}));
+
+const WidgetStatGrid = styled(Grid)(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  gap: theme.spacing(0.5),
+  marginBottom: theme.spacing(1.5),
+}));
+
+interface HealthIndicatorWidgetProps {
+  accountListId: string;
+  showHealthIndicator: boolean;
+  setShowHealthIndicator: Dispatch<SetStateAction<boolean>>;
+}
+
+export const HealthIndicatorWidget: React.FC<HealthIndicatorWidgetProps> = ({
+  accountListId,
+  showHealthIndicator,
+  setShowHealthIndicator,
+}) => {
+  const { t } = useTranslation();
+
+  const { data, loading } = useHealthIndicatorWidgetQuery({
+    variables: {
+      accountListId,
+    },
+  });
+
+  useEffect(() => {
+    setShowHealthIndicator(!!data?.healthIndicatorData.length);
+  }, [data]);
+
+  if (!showHealthIndicator) {
+    return null;
+  }
+
+  const currentStats =
+    data?.healthIndicatorData[data?.healthIndicatorData.length - 1];
+
+  return (
+    <AnimatedCard sx={{ height: '100%' }}>
+      <CardHeader
+        title={t('MPD Health Indicator')}
+        titleTypographyProps={{
+          style: { fontSize: '1rem' },
+        }}
+      />
+      <StyledCardContent>
+        <StyledBox>
+          <Typography variant="h4" color="primary" width={'55px'}>
+            {currentStats?.overallHi}
+          </Typography>
+          <Box width={'calc(100% - 55px)'}>
+            <Typography
+              component="div"
+              color="primary"
+              sx={{ marginBottom: 1 }}
+            >
+              {t('Overall Health Indicator')}
+            </Typography>
+            <StyledProgress
+              loading={loading}
+              primary={
+                currentStats?.overallHi ? currentStats.overallHi / 100 : 0
+              }
+              barHeight={20}
+            />
+          </Box>
+        </StyledBox>
+
+        <Grid container>
+          <WidgetStat
+            loading={loading}
+            stat={currentStats?.ownershipHi}
+            statName={t('Ownership')}
```

Comment:
I believe we wanted a tooltip for each stat that displays a one-sentence definition of the stat. Shawn or Toncho may have suggestions on the wording, but I think we can start with these from the doc:

* Ownership: % of self-raised funds over total funds
* Success: % of self-raised funds over support goal
* Consistency: % of months with a positive account balance
* Depth: trend of local partners